ibis/backends/duckdb/tests/test_client.py

```diff
@@ -78,16 +78,10 @@ def test_cross_db(tmpdir):
 
     con2.attach(path1, name="test1", read_only=True)
 
-    with pytest.warns(FutureWarning):
-        t1_from_con2 = con2.table("t1", schema="main", database="test1")
+    t1_from_con2 = con2.table("t1", database="test1")
     assert t1_from_con2.schema() == t2.schema()
     assert t1_from_con2.execute().equals(t2.execute())
 
-    with pytest.warns(FutureWarning):
-        foo_t1_from_con2 = con2.table("t1", schema="foo", database="test1")
```

Comment:
This shouldn't be removed, but refactored to use either a dotted path or a tuple of path elements, like `"test1.foo"`

Reply:
fixed, I added it back using a dotted path.